pkg/apis/openstack/validation/cloudprofile.go

```diff
@@ -192,3 +189,47 @@ func ValidateLoadBalancerClasses(lbClass api.LoadBalancerClass, fldPath *field.P
 
 	return allErrs
 }
+
```

Comment:
Nit: Comment missing for exported function

Reply:
done. Thanks :)